modules/oe_whitelabel_paragraphs/tests/src/Kernel/Paragraphs/ColorSchemeTest.php

```diff
@@ -79,10 +79,8 @@ public function testColorSchemeInParagraphs(): void {
       $html = $this->renderParagraph($paragraph);
       $crawler = new Crawler($html);
 
-      $element = $crawler->filter($data['wrapper_selector'] . '.foo-bar.text-color-default');
+      $element = $crawler->filter($data['wrapper_selector'] . '.foo-bar');
       $this->assertCount(1, $element, sprintf('Element "%s" has color scheme applied.', $data['wrapper_selector']));
-      $has_background = (bool) $crawler->filter('.bg-default')->count();
-      $this->assertTrue($data['has_background'] === $has_background, sprintf('Element "%s" does not have ".bg-default" class.', $data['wrapper_selector']));
```

Comment:
had to delete the negative testing because of the increased complexity of the injectColorScheme "API"